src/daemon.rs

```diff
@@ -113,9 +114,13 @@ impl Daemon {
         &self,
         txid: &Txid,
         blockhash: Option<BlockHash>,
-    ) -> Result<json::GetRawTransactionResult> {
+    ) -> Result<Value> {
+        let args = blockhash.map_or_else(
```

Comment:
TBH, I find `match` clearer than `map_or_else`, the additional boilerplate is minimal.

Reply:
Agree, but it seems that passing `null` blockhash works as well :)
https://github.com/bitcoin/bitcoin/blob/f8b20fd35b0eb42d28afb302a070e5d9bc21a99f/src/rpc/rawtransaction.cpp#L180

Reply:
Ah, so `None` gets serialized as `null`, right?

Reply:
Yep :)